### DISCUSSION TITLE
Vercel — Commit author needs to be GitHub account - Error

### DISCUSSION THREAD

**Discussion by victorkosmo** [vercel/community#4855] [Bug Report]
### Page to Investigate

1

### Steps to Reproduce

Hello, I was trying to upgrade to Pro and could not do this in my default "team" - so I created new team on Trial, then started to move projects there.

I noticed that some of them stopped updating automatically from GitHub repo updates. Projects that were able to pull latest updates were stored in my previous "team" on hobby plan. All projects that were moved to new one getting this not on GitHub

**Vercel — Commit author needs to be GitHub account**

I thought that moving all my projects to new team would solve issue - but It did not helped. I revoked GitHub auth access - and did not helped either. My mail and username are identical on GitHub and Vercel platforms.

Please advise, thanks

**Comment by lenmerkel** [comment]
I have the same issue. After creating a new Pro (Trial) account and moving projects from my Hobby account, they no longer deploy on pushes to the repo. Deploying from my own system with the CLI ("vercel --prod") works OK.

Please advise how to fix. Thanks.

**Edit**: Please see my follow-up comment for solution.

**Comment by lenmerkel** [comment]
OK - problem solved. I needed to set a commit email address in my local Git setup, where that email address is associated with my GitHub account. Now, commits are auto-deploying again. Odd that this wasn't a problem when my GitHub repo was connected to my Hobby account, but is a problem after transferring to a Pro account.

I found out by seeing the Vercel error in my repo's commit history:

![GitHub](https://github.com/vercel/community/assets/14983777/ee9ef2cf-9260-4015-a825-6b3385c15612)

That led me to the relevant GitHub article [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address).
